HT4_part1.py

```diff
@@ -0,0 +1,50 @@
+# import modules random and string
+import random
+import string
+
+def random_dicts():

```

Comment:
I think we need to parametrize this function, how many dicts should this function create, what is the range of keys for dicts, what is the range of values for these keys. It is necessary for reusing this function in other modules of the future program product